### DISCUSSION TITLE
Thrustmaster Pain

### DISCUSSION THREAD

**Discussion by frogmonkey2** [flybywiresim/aircraft#4138] [Help] [answered]
I have tried every new trick in the book. The 'latest' FBW version has eliminated the work folders (Thrustmaster INI. file) that was utilized for set up. That's gone now with the new FlyPad. So I 'attempted' to set up using my throttle and the pad. All's fine with the exception of the 'reversers' where it just 'stops'? Had a heck of a time with setting up the Thrustmaster with your mod... my only complaint so far... no material documentation for reference of ?... maybe a plug and play option?... most Thrustmeter's don't change? as far as throttle use that is. (using the experimental mod)

**Comment by Mickepilot** [comment]
Well, I also had initial problem to get the "FlyPad" calibration working ok. Important is to activate "Reverser On Axis = On".
It's optional if you like two do the settings separate for the Left and Right throttle. It that case activate "Dual Axis = On".
You have to imagine going from throttle rear position (-1 =Reverse) towards max front position (1 = TO/GA).
All positions in between is (with the help of "Start" and "End" settings) like small detents, hiding any inprecision in the Thrustmaster TCA Quadrant sensor position or calibration.
For the reverser values, you have to fiddle a little bit... why not try my settings?:
REVERSE: Start= -1 End= -0.98
REV.IDLE: Start= -0.7 End= -0.65
IDLE: Start= -0.56 End= -0.44

**Reply by Paytheon** [reply]
Here's a youtube video on how to setup the Thrustmaster AB Joystick and Throttle by our old friend 320 Sim Pilot: https://www.youtube.com/watch?v=OCCJOQyWyq8

**Reply by frogmonkey2** [reply]
Thanks, friends, I will try your settings. These things you tell me are what the FBW team should tell us after each 'change' instead of chasing around information on every sim site on the internet just to get a answer... ya know?

**Reply by frogmonkey2** [reply]
your DATA set value range is not far off from the user below here... each work, they are off by a single point only... very close REVERSE: Start= -1 End= -0.98
REV.IDLE: Start= -0.7 End= -0.65
IDLE: Start= -0.56 End= -0.44

reverse_low = -1.000000
reverse_high = -0.980000
reverse_idle_low = -0.720000
reverse_idle_high = -0.620000
idle_low = -0.550000
idle_high = -0.470000

**Comment by Annolago** [comment]
Are you sure that your problem is not due to the fact that the subfolder names of the A32nx folder has been changed?
This means that the throttle.ini subfolder has also been changed. So you need to look in your company folder - see what the name is and then check and see if the settings are in a new folder in the packages subfolder. 
To make it a bit clearer - before the folder for the installation was in the community folder and was named "a32nx" - that has been changed to "flybywire-aircraft-a320-neo" - the same applies for the packages folder. So go look in your packages folder and see if you now have a new folder with the new name.
Hope I did not misunderstand and this will help

**Reply by frogmonkey2** [reply]
Yes, I realized the change in path. Yes appreciated help.

**Comment by mojoair** [comment, marked as answer]
As others have mentioned, the folder name has changed to  "flybywire-aircraft-a320-neo". 

Once you manipulate/change any of the value of the calibration on the EFB you must click apply and save. Once you do that, a ThrottleConfiguration.ini file will be generated in this new work folder. 

You can try the following in the new throttle config file as a starting point. My settings are with a fully linear with 0 detent in MSFS settings.


[throttle_common]
reverse_on_axis = true

[throttle_axis_1]
reverse_low = -1.000000
reverse_high = -0.980000
reverse_idle_low = -0.720000
reverse_idle_high = -0.620000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.060000
flex_mct_low = 0.410000
flex_mct_high = 0.590000
toga_low = 0.970000
toga_high = 1.000000

[throttle_axis_2]
reverse_low = -1.000000
reverse_high = -0.980000
reverse_idle_low = -0.720000
reverse_idle_high = -0.620000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.060000
flex_mct_low = 0.410000
flex_mct_high = 0.590000
toga_low = 0.970000
toga_high = 1.000000

**Reply by frogmonkey2** [reply]
I will try your settings, sir! THX

**Reply by frogmonkey2** [reply]
those data set points are perfect sir thank you... today I fly again... hopefully... no more major confusing updates hopefully... 🤩

**Reply by mojoair** [reply]
Perfect. Glad it helped. Please mark the post as answered so others can hopefully see the solution as well :). Thanks.

**Comment by frogmonkey2** [comment]
besides your value sets, I got the author of the new flyPad Andreas Gunther to respond to my queries which were admittedly getting slightly frustrated and out of my wit's end, however, I finally got his response after being chastised for my 'attitude' however, I DID get answers and DID receive updates as to what where and when. Whereas before as you and I know we just opened the mod to 'find' it didn't work... again... ya know?

**Reply by frogmonkey2** [reply]
Andreas Gunthers DATA set, you need to check and place your own DATA set values here - the ([THROTTLE_AXIS_1] & 2) and rearrange them to your DATA his is different as he's the coder
*Andreas Guther
Mine looks like this (you cannot use it 1:1 because I use a modded version of the TCA):
[THROTTLE_COMMON]
REVERSE_ON_AXIS = true
[THROTTLE_AXIS_1]
REVERSE_LOW = -1.00
REVERSE_HIGH = -0.98
REVERSE_IDLE_LOW = -0.80
REVERSE_IDLE_HIGH = -0.60
IDLE_LOW = -0.47
IDLE_HIGH = -0.41
CLIMB_LOW = 0.25
CLIMB_HIGH = 0.35
FLEX_MCT_LOW = 0.70
FLEX_MCT_HIGH = 0.90
TOGA_LOW = 0.98
TOGA_HIGH = 1.00
[THROTTLE_AXIS_2]
REVERSE_LOW = -1.00
REVERSE_HIGH = -0.98
REVERSE_IDLE_LOW = -0.80
REVERSE_IDLE_HIGH = -0.60
IDLE_LOW = -0.47
IDLE_HIGH = -0.41
CLIMB_LOW = 0.25
CLIMB_HIGH = 0.35
FLEX_MCT_LOW = 0.70
FLEX_MCT_HIGH = 0.90
TOGA_LOW = 0.98
TOGA_HIGH = 1.00

**Comment by gboivin53** [comment]
I have not got the thrustmaster throttle to work with the experimental version. No response on the throttles. But switch to stable or the custom one. The Thrustmaster works as it should. I hope that there will be  instruction as the beta progresses.

**Comment by CptBayer** [comment]
Hi,


my settings working wonderfull...

[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.0
DetentReverseFull = -1.00
DetentReverseIdle = -0.69
DetentIdle = -0.50
DetentClimb = -0.02
DetentFlexMct = 0.49
DetentTakeOffGoAround = 1.00

[throttle_common]
reverse_on_axis = true

[throttle_axis_1]
reverse_low = -0.700000
reverse_high = -1.000000
reverse_idle_low = -0.560000
reverse_idle_high = -0.690000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.020000
flex_mct_low = 0.470000
flex_mct_high = 0.510000
toga_low = 0.980000
toga_high = 1.000000

[throttle_axis_2]
reverse_low = -0.700000
reverse_high = -1.000000
reverse_idle_low = -0.560000
reverse_idle_high = -0.690000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.020000
flex_mct_low = 0.470000
flex_mct_high = 0.510000
toga_low = 0.980000
toga_high = 1.000000

greets Klaus

**Reply by gboivin53** [reply]
I'll try this. I'll copy to the init file. I don't show any throttle movement using the EFB. MSFS show that its working fine.

**Reply by CptBayer** [reply]
...tell me if you had success !

**Reply by frogmonkey2** [reply]
remember to just create the file 1st... those settings-(CptBayer)-are close to mine 
____________________________________________________________________________________
NEW /MINE - (these I got from another user as I was initially having issues setting up, they appear to also work fine for me... I wish they-(FBW)- could just put out a universal ini. file for the TCA as most of us use it)

[throttle_common]
reverse_on_axis = true

[throttle_axis_1]
reverse_low = -1.000000
reverse_high = -0.980000
reverse_idle_low = -0.720000
reverse_idle_high = -0.620000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.060000
flex_mct_low = 0.410000
flex_mct_high = 0.590000
toga_low = 0.970000
toga_high = 1.000000

[throttle_axis_2]
reverse_low = -1.000000
reverse_high = -0.980000
reverse_idle_low = -0.720000
reverse_idle_high = -0.620000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.060000
flex_mct_low = 0.410000
flex_mct_high = 0.590000
toga_low = 0.970000
toga_high = 1.000000

______________________________________________________________________
OLD - Example for Thrustmaster TCA Quadrant Airbus Edition (2)

[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.5
DetentReverseIdle = -0.90
DetentReverseFull = -1.00
DetentIdle = -0.42
DetentClimb = 0.06
DetentFlexMct = 0.53
DetentTakeOffGoAround = 1.00

[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.0
DetentReverseFull = -1.00
DetentReverseIdle = -0.70
DetentIdle = -0.44
DetentClimb = -0.10
DetentFlexMct = 0.53
DetentTakeOffGoAround = 1.00

**Comment by gboivin53** [comment]
That made it worse- went from reverse then next indent was climb .  There has got to be a better way than trial and error. I got the thrust master specifically for the a320.  I have the honeycomb throttle also but haven’t tried that for the a320.

Sent from my iPhone

On Apr 5, 2021, at 6:29 PM, frogmonkey2 ***@***.***> wrote:

﻿

remember to just create the file 1st... those settings-(CptBayer)-are close to mine

________________________________

NEW /MINE - (these I got from another user as I was initially having issues setting up, they appear to also work fine for me... I wish they just put out a universal for the TCA as most of us use it)

[throttle_common]
reverse_on_axis = true

[throttle_axis_1]
reverse_low = -1.000000
reverse_high = -0.980000
reverse_idle_low = -0.720000
reverse_idle_high = -0.620000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.060000
flex_mct_low = 0.410000
flex_mct_high = 0.590000
toga_low = 0.970000
toga_high = 1.000000

[throttle_axis_2]
reverse_low = -1.000000
reverse_high = -0.980000
reverse_idle_low = -0.720000
reverse_idle_high = -0.620000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.060000
flex_mct_low = 0.410000
flex_mct_high = 0.590000
toga_low = 0.970000
toga_high = 1.000000

________________________________

OLD - Example for Thrustmaster TCA Quadrant Airbus Edition (2)

[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.5
DetentReverseIdle = -0.90
DetentReverseFull = -1.00
DetentIdle = -0.42
DetentClimb = 0.06
DetentFlexMct = 0.53
DetentTakeOffGoAround = 1.00

another user

[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.0
DetentReverseFull = -1.00
DetentReverseIdle = -0.70
DetentIdle = -0.44
DetentClimb = -0.10
DetentFlexMct = 0.53
DetentTakeOffGoAround = 1.00

**Reply by frogmonkey2** [reply]
I'm with you there. When I started with my throttle it was a simple input ini. file. Since the pad was introduced it's been nothing but a hassle. 

[throttle_common]
reverse_on_axis = true

[throttle_axis_1]
reverse_low = -0.700000
reverse_high = -1.000000
reverse_idle_low = -0.560000
reverse_idle_high = -0.690000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.020000
flex_mct_low = 0.470000
flex_mct_high = 0.510000
toga_low = 0.980000
toga_high = 1.000000

[throttle_axis_2]
reverse_low = -0.700000
reverse_high = -1.000000
reverse_idle_low = -0.560000
reverse_idle_high = -0.690000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.020000
flex_mct_low = 0.470000
flex_mct_high = 0.510000
toga_low = 0.980000
toga_high = 1.000000

**Reply by frogmonkey2** [reply]
this may also be old now... my TM quads down due to a sensor out... just got it and walla no workie after a few weeks

**Comment by kolyadkl** [comment]
Hi! Anybody know can I use Thrustmaster with new stable ver 0.6.0? If yes, how?
